### PR TITLE
Add blogpost about FAS verification automation

### LINKED ISSUE #461: Create short blogpost about our new allowlisting mechanism in packit service
- [ ] describe the new mechanism 
- [ ] include screenshots (you can use packit/notifications#429)

### PR DESCRIPTION
**Pull request by lbarcziova**
Fixes #461 
Preview:
image

### CODE REVIEW

**Review by jpopelka — APPROVED** [review]
nice

File: content/posts/fas-verification-automation.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+So how does the verification work now?
+
+For each new GitHub installation, we first check whether there isn't a FAS account with the same login as the one
+that triggered the installation. If we find such an account, we check whether the `GitHub Username` in this FAS account matches
+the Github login of the one that triggered the installation. To get the information about the FAS accounts, we
+use the [`fasjson-client` library](https://github.com/fedora-infra/fasjson-client).
+If this check doesn't prove any match, we create an issue in the
+`packit/notifications` repository as previously. This is what it looks like:
+
+![Issue in packit/notifications](/images/notifications-repo-issue.png)
```
Comment (lachmanfrantisek) [inline content/posts/fas-verification-automation.md:31]: What about removing the header so we have a white background and the issue text is better readable?

**Review by lachmanfrantisek — APPROVED** [review]
Really nice!

File: content/posts/fas-verification-automation.md (hunk 1 above)
Comment (lbarcziova) [reply content/posts/fas-verification-automation.md:31]: firstly I wanted to emphasize that the issue is in packit/notifications repo, but it is described in the text well, so you are right, it looks better without it, thanks!

**Pull request merged**